### ISSUE TITLE
CLI flag to disable pre-release-replacements

### ISSUE THREAD

**Issue by DJDuque** [crate-ci/cargo-release#810]
I read through the reference, and I think this is not currently possible, right?

It would be helpful for workflows like the one I described in #809. And given that there are already flags to disable e.g. publish, tag, etc. maybe it makes sense to have this one as well.

**Comment by epage** [comment]
Yes, `cargo release` is a "do everything" command and the individual "steps" are generally how we expect people to customize behavior.  With `cargo release tag` and other commands, I'm unsure how much value add there is with a "skip replacements" flag.

**Comment by DJDuque** [comment]
This makes sense. Now that I understand how to customize the behavior with individual steps, I don't have a strong opinion about this.

On the other hand, there are already flags like `--no-publish`, `--no-tag`, `--no-push` (and you can also not bump the version by not specifying it). Basically we can currently skip any step except replace. Just for consistency, I think it can make sense to allow this.

It would also probably make a more intuitive experience for users to know that all steps are "equal". You can run them individually, or you can exclude them individually.
